tests/data/config.yaml

```diff
@@ -1,4 +1,5 @@
 # Configuration for i18n workflow.
```

Comment:
Maybe a different comment on the test file, since it has weird non-default values in it?